From 780803a648610de6f727502ec392bb30504e153a Mon Sep 17 00:00:00 2001
From: reaper <theunbounddeveloper@outlook.com>
Date: Tue, 4 Aug 2026 17:41:22 -0400
Subject: [PATCH 322/322] reaper v2.1.5: version bump (MULTISERVICE_WAN PPPoE
 baby-jumbo MTU fix + self-recovering first-boot credential page + rtrafd
 JFFS/USB store persistence)

---
 release/src-rt/version.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/release/src-rt/version.conf b/release/src-rt/version.conf
index fea49c5d3d..6df09844b6 100644
--- a/release/src-rt/version.conf
+++ b/release/src-rt/version.conf
@@ -4,6 +4,6 @@ KERNEL_VER_AM=$(subst .,,$(KERNEL_VER))
 FS_VER_AM=$(subst .,,$(FS_VER))
 
 SERIALNO=102.8
-EXTENDNO=Reaper_v2.1.4
+EXTENDNO=Reaper_v2.1.5
 
 RCNO=0
-- 
2.25.1

